src/data/content/community.yaml

```diff
@@ -62,19 +62,7 @@ components:
         icon: sprites.svg#it-horn
         size: sm
         color: white
-      chips:
-        - label: Accessibilità
-          url: "#"
-          color: primary
-          size: lg
-        - label: Ricerca Utente
-          url: "#"
-          color: primary
-          size: lg
-        - label: Sviluppo front end
-          url: "#"
-          color: primary
-          size: lg
+      tags: [Accessibilità,Ricerca utente,Sviluppo front end]
```

Comment:
This is a great example on how abstracting a concept (the tag in this case), allows us to keep just the content in the YAML file and have the rest handled automatically dramatically simplifies things

There's no pseudo-HTML or presentation side mixed here. 

 :clap: